files/usr/share/cinnamon/cinnamon-settings/xlet-settings.py

```diff
@@ -262,6 +262,10 @@ def load_instances(self):
                                     continue
                                 new_opt_data[translate(self.uuid, option)] = opt_data[option]
                             settings_map[setting][key] = new_opt_data
+                        elif key in "columns":
```

Comment:
Hello, @NikoKrause.

Replace that entire block of code that you added with the following...

```python
elif key in "columns":
    col_data = settings_map[setting][key]

    for col_idx in range(0, len(col_data)):
        try:
            settings_map[setting][key][col_idx]["title"] = translate(self.uuid, col_data[col_idx]["title"])
        except:
            pass
```

...that should make it work.

Reply:
Where's the difference?

Reply:
> Where's the difference?

I have no idea. I stopped reading your code when I saw the use of the `_()` function (which shouldn't be used) and I "mashed up" some code for you to test.